app/src/main/res/layout/activity_example1.xml

```diff
@@ -18,5 +18,5 @@
         android:text="Click"
         android:textAppearance="@style/Roboto_Regular"
         tools:ignore="HardcodedText"
-        />
+    />
```

Comment:
has the codestyle for this changed?

Reply:
I auto-formatted but it seems like there's some code-style to fix still